Feat: Adicionado parâmetro de content-type para mock (#15)
PopcodeMobile/popnetwork: CHANGELOG.md

## [1.2.0] - Add new parameter to allow response from mock with Plain Text

> - Add `MockContentTypeEnum` parameter to `ApiManager` to mock sends Plain Text response correctly

Reviewer: ```suggestion
- Add `MockContentTypeEnum` parameter to `ApiManager` so mock can return Plain Text response correctly
```

- Parse String or Json according to the new parameter in mock
- Send correct content-type header according to the new parameter in mock

> - Update version dio to 5.8.0+1

Reviewer: ```suggestion
- Update Dio version to 5.8.0+1
```

> - Reduce delay to 500 milliseconds

Reviewer: ```suggestion
- Reduce delay on `MockReplyParams` to 500 milliseconds
```
